qiita_ware/private_plugin.py

```diff
@@ -402,9 +402,6 @@ def list_remote_files(job):
             job._set_error(traceback.format_exception(*exc_info()))
         else:
             job._set_status('success')
-        finally:
-            # making sure to always delete the key so Qiita never keeps it
```

Comment:
?

Reply:
This is the bug we found that was preventing listing files via scp for study 12294; basically `list_remote` already removes the key and this fails cause the file never exists when we reach this point.

Reply:
ah, thanks!